doc/sections/2.Background.tex

```diff
@@ -0,0 +1,165 @@
+\section{Background Theory}
+\label{sec:background}
+
+\subsection{ABFT matrix multiplication}
+
+The basic idea behind Algorithm-Based Fault Tolerant (ABFT) matrix multiplication
+is to use checksums to verify the correctness of the computation.
+These are the steps we perform:
+
+\begin{enumerate}
+  \item \textbf{Checksum Calculation}:
+    Let \( A \) be an \( m \times n \) matrix,
+    \( B \) be an \( n \times q \) matrix,
+    and \( C \) be the resulting \( m \times q \) matrix
+    from the multiplication \( C = A B \).
+    Before performing the matrix multiplication,
+    we calculate the column checksum of \( A \) and the row checksum of \( B \):
+
+    \[
+      \textnormal{Column checksum of } A: \quad
+      \mathbf{c}_A =
+      \left[
+        \sum_{i=1}^{n} a_{i1},
+        \sum_{i=1}^{n} a_{i2},
+        \ldots, \sum_{i=1}^{n} a_{in}
+      \right]
+    \]
+
+    \[
+      \textnormal{Row checksum of } B: \quad
+      \mathbf{r}_B = \left[ \sum_{j=1}^{n} b_{1j}, \sum_{j=1}^{n} b_{2j}, \ldots, \sum_{j=1}^{n} b_{nj} \right]
```

Comment:
Metterei una T in apice a r_B, per far capire che è trasposto... se no dici che r_B è una row checksum, dici che le row checksum sono vettori colonna, ma dici che r_B è un vettore riga